make_release.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+set +x
+
+if [[ $# -eq 0 ]] ; then
+    echo "Please supply a 'version' as argument."
```

Comment:
```suggestion
    echo "Please supply a 'version' as argument (e.g., ./make_release.sh 1.0.0)."
```

or is it with a preceding "v"?

Reply:
The "v"-prefix is only for the git tag, not for the scripts.